src/artemis/experiment_plans/optimise_attenuation_plan.py

```diff
@@ -58,6 +79,87 @@ def arm_devices(xspress3mini, zebra):
     yield from arm_zebra(zebra)
 
 
+def deadtime_is_transmission_optimised(
+    direction, deadtime, deadtime_threshold, transmission
+) -> tuple[bool, bool]:
+    flip_direction = False
+    if direction:
+        if deadtime >= deadtime_threshold:
+            flip_direction = True
+            return False, flip_direction
+        else:
+            # The 0.9 is hardcoded in GDA
+            if transmission >= 0.9:
+                return True, flip_direction
```

Comment:
Must: Are we sure this is right? It looks wrong to me because if the deadtime is less than threshold and the transmission less than 90% the function returns nothing